src/doc/trpl/guessing-game.md

```diff
@@ -239,7 +239,7 @@ use std::rand;
 fn main() {
     println!("Guess the number!");
 
-    let secret_number = (rand::random::<uint>() % 100u) + 1u;
+    let secret_number = (rand::random::<u32>() % 100u) + 1u;
```

Comment:
Should I make another commit and issue a separate pull request?

I'm not fully sure of the procedure.